robottelo/test.py

```diff
@@ -169,14 +309,78 @@ def log_method_name(self, prefix=None):
             self._testMethodName,
         ))
 
+    def assertNotRaises(self, expected_exception, callableObj=None,
```

Comment:
I was following unittest style - for assertRaises it's called `callableObj`, as both assertions are pretty similar i wanted to keep interface as similar as possible. Any advantages in following different style?